tests/specs/constructor.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+var test = require('tape')
+
+var dbFactory = require('../utils/db')
+var Store = require('../../')
+
+test('constructs a store object', function (t) {
+  t.plan(2)
+
+  var db = dbFactory()
+  var store = new Store(db)
+
+  t.is(typeof store, 'object', 'is object')
+  t.is(store.db, db, 'exposes db')
```

Comment:
ah, I'm sorry I didn't make this clear. PouchDB will be a hard dependency of `pouchdb-hoodie-store`. The API will be 

```js
store = new Store(dbName, options)
```

You don't pass a PouchDB instance to `new Store()`. Instead, in the `Store` constructor, initialise the `db` instance based on the passed `name`. 

Makes sense?